src/main/java/org/example/Main.java

```diff
@@ -1,7 +1,167 @@
 package org.example;
 
+import java.util.Scanner;
+import java.util.Set;
+
 public class Main {
```

Comment:
You can name this class the actuall app name, not 'Main' but something like TicTacToeGame or something shorter. So that name indicates what the game is about